cvat/apps/engine/views.py

```diff
@@ -83,6 +83,26 @@
 
 _UPLOAD_PARSER_CLASSES = api_settings.DEFAULT_PARSER_CLASSES + [MultiPartParser]
 
+def _tus_chunk_action(*, detail: bool, suffix_base: str):
```

Comment:
I suggest putting this function to `view_utils.py`

Reply:
Alright, done.

Reply:
This caused an import loop, so I moved some stuff around to fix it.